extensions/OpenIddict.Validation/OpenIddict.Validation.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\packages.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>OpenIdDict reference tokens validation middleware for ASP.NET Core.</Description>
+    <Authors>Kévin Chalet,Chino Chang</Authors>
+    <PackageTags>aspnetcore;authentication;jwt;openidconnect;openiddict;security</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\OpenIddict.Core\OpenIddict.Core.csproj" />
+    <ProjectReference Include="..\..\src\OpenIddict.Models\OpenIddict.Models.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AspNet.Security.OAuth.Validation" Version="$(AspNetContribOauthExtensionsVersion)" />
+    <PackageReference Include="AspNet.Security.OpenIdConnect.Server" Version="$(AspNetContribOpenIdServerVersion)" />
```

Comment:
Consider removing that. Feel free to hardcode the Data Protection purposes instead of using `nameof`.

Reply:
@PinpointTownes `AspNet.Security.OpenIdConnect.Server` also provides the `AuthenticationProperties.GetProperty()` method which is required in the handler